<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ANURAG24" failures="0" tests="2" name="com.crm.vtiger.TestScript.TC_42_searchLeadByClickingOnWebsite_Test" time="38.306" errors="1" timestamp="09 May 2021 02:33:17 GMT" skipped="0">
  <testcase classname="com.crm.vtiger.TestScript.TC_42_searchLeadByClickingOnWebsite_Test" name="creatWebsite" time="38.303"/>
  <testcase classname="com.crm.vtiger.TestScript.TC_42_searchLeadByClickingOnWebsite_Test" name="configAS" time="0.003">
    <error message="Cannot invoke &quot;java.sql.Connection.close()&quot; because &quot;com.crm.vtiger.GenericUtilils.DataBaseUtilities.con&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.sql.Connection.close()" because "com.crm.vtiger.GenericUtilils.DataBaseUtilities.con" is null
at com.crm.vtiger.GenericUtilils.DataBaseUtilities.closeDb(DataBaseUtilities.java:36)
at com.crm.vtiger.GenericUtilils.BaseTest.configAS(BaseTest.java:99)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:425)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- configAS -->
</testsuite> <!-- com.crm.vtiger.TestScript.TC_42_searchLeadByClickingOnWebsite_Test -->
